agent/lib_aws_sdk_php.c

```diff
@@ -15,9 +15,16 @@
 #include "fw_support.h"
 #include "util_logging.h"
 #include "nr_segment_message.h"
+#include "nr_segment_external.h"
 #include "lib_aws_sdk_php.h"
 
 #define PHP_PACKAGE_NAME "aws/aws-sdk-php"
+#define AWS_ARN_REGEX "(arn:(aws[a-zA-Z-]*)?:lambda:)?" \
```

Comment:
Please change to AWS_ARN_LAMBDA_REGEX so we can differentiate when we get to other services that need ARNs as well.

Reply:
changed in [5c45a0e](https://github.com/newrelic/newrelic-php-agent/pull/1023/commits/5c45a0ef346dbb2bf7e1b28a8ed0de4cbabcbb3f)